base_layer/core/src/validation/header/header_full_validator.rs

```diff
@@ -70,16 +78,19 @@ impl<B: BlockchainBackend> HeaderChainLinkedValidator<B> for HeaderFullValidator
     ) -> Result<AchievedTargetDifficulty, ValidationError> {
         let constants = self.rules.consensus_constants(header.height);
 
-        check_not_bad_block(db, header.hash())?;
-        check_blockchain_version(constants, header.version)?;
-        check_height(header, prev_header)?;
-        check_prev_hash(header, prev_header)?;
+        // dont run these checks for blocks we have whitelisted
+        if !WHITELISTED_HEADERS.contains(&header.hash().to_hex().as_str()) {
+            check_not_bad_block(db, header.hash())?;
+            check_blockchain_version(constants, header.version)?;
+            check_height(header, prev_header)?;
+            check_prev_hash(header, prev_header)?;
```

Comment:
Maybe we need to be more specific here and only exlude the tests we know would fail for the whitelisted block, not all of it.

Reply:
There is no point doing any validation if the whole header is whitelisted